loleaflet/src/control/Control.NotebookbarWriter.js

```diff
@@ -124,33 +124,88 @@ L.Control.NotebookbarWriter = L.Control.Notebookbar.extend({
 								}
 							]
 						} : {},
-					hasPrint ?
+					hasRevisionHistory ?
 						{
-							'id': 'Section4',
+							'id': 'Section5',
 							'type': 'toolbox',
 							'text': '',
 							'enabled': 'true',
 							'children': [
 								{
-									'id': 'print',
+									'id': 'rev-history',
 									'type': 'bigtoolitem',
-									'text': _UNO('.uno:Print', 'text'),
-									'command': '.uno:Print'
+									'text': _('See history'),
+									'command': '.uno:rev-history'
```

Comment:
shareas and rev-history are now duplicated: we have 2 bigtoolitems and 2 small items

Reply:
strange. But I can use the small items instead of the big one isn't it? or is there some technical issue?

cause than I will make a separate request for ALL NB home tabs.

Reply:
it is because you added new items but not deleted old ones from the JSON :)